src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using RegressionGames;
+using TMPro;
+using UnityEngine;
+using RegressionGames.StateRecorder.BotSegments.Models;
+
+/**
+ * <summary>UI for managing operations performed on the user's set of Bot Sequences</summary>
+ */
+public class RGSequenceManager : MonoBehaviour
+{
+    public GameObject sequencesPanel;
+
+    public GameObject sequenceCardPrefab;
+
+    private IList<BotSequence> _sequences;
+
+    private static RGSequenceManager _this;
+
+    private const string SEQUENCES_PATH = "Assets/RegressionGames/Resources/BotSequences";
+
+    public static RGSequenceManager GetInstance()
+    {
+        return _this;
+    }
+
+    private void Awake()
+    {
+        _sequences = new List<BotSequence>();
+
+        LoadSequences();
+
+        _this = this;
+        DontDestroyOnLoad(_this.gameObject);
+    }
+
+    /**
+     * <summary>Loads all Sequence json files within the Unity project</summary>
+     */
+    public void LoadSequences()
+    {
+        if (Directory.Exists(SEQUENCES_PATH))
+        {
+            // ensure we arent' appending new sequences on to the previous ones
+            ClearExistingSequences();
+
+            IEnumerable<string> sequenceFiles = Directory.EnumerateFiles(SEQUENCES_PATH, "*.json");
+            foreach (string fileName in sequenceFiles)
```

Comment:
Before this, could we map the files to their loaded bot sequences and then sort by last modified in descending order? I think as we have more and more sequences, it will be useful to have the most recently modified one at the front. Thoughts?

```csharp
var sequences = sequenceFiles
            .Select(f => BotSequence.LoadSequenceJsonFromPath(f))
            .OrderByDescending(seq => seq.lastModified)
            .ToList();
```

Reply:
Works for me! Done